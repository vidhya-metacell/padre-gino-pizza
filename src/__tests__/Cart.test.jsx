import { expect, test } from "vitest";
import { render } from "@testing-library/react";
import Cart from "../Cart";


test("test cart is loaded empty snapshot", () => {
  const { asFragment } = render(() => <Cart cart={[]} />);
  expect(asFragment()).toMatchInlineSnapshot(`<DocumentFragment />`);
})