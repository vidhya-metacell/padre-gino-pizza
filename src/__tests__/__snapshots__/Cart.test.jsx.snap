// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`test cart is loaded empty snapshot 1`] = `<DocumentFragment />`;
